Yoast/wordpress-seo
# Fix Create GitHub pre release task

## Summary


This PR can be summarized in the following changelog entry:

* [non-user-facing] Fixed requests to Github API for prereleases.

## Relevant technical choices:

* Creating and pushing a tag was not necessary, as investigated for #14043 . The POST request to release already creates the task.

## Test instructions

This PR can be tested by following these steps:

* Fork `wordpress-seo` to your personal repos.
* For testing, uncomment line 49 in `github-pre-release.js`, and replace `{ YOUR PERSONAL REPO }` with, well, your personal repo.
* If you haven't done so, copy `.env.example` to `.env`.
* [Create an access token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) on your fork, and copy it, and paste it after `GITHUB_TOKEN` in your `.env`.
* For ease of testing (but this is up to you), comment line 40 (`const changelog = openChangelogEditor( grunt );`) and replace it with `const changelog = ""`.
* Replace `const pluginVersion = `etc... with `const pluginVersion = "testingVersion"`
* Run `grunt github-pre-release`. On the releases tab of your fork, there should now be a pre-release with the name (and tag) `vtestingVersion`.
* Running the script again will throw an error: the tag already exists. Create a different name for running it again.
* `grunt.config.set( "rc.github.url", constructedZipUrl );` should set a valid ZIP URL on the grunt config, log it by adding `console.log( grunt.config.get( "rc.github.url" ) );`.

## UI changes

## Documentation

## Quality assurance


Fixes #14058 
Fixes #14043
